### DISCUSSION TITLE
Wrong explanation in FlatGeobuf Spatial Index

### DISCUSSION THREAD

**Discussion by mrmichaelf** [cloudnativegeo/cloud-optimized-geospatial-formats-guide#100] [General]
It' written:
"This Crunchy Data blog post has helpful examples for **why sorting input is important**."
The blog post says:
"Pre-ordering your rows **randomly** will result in a more balanced tree and measurable improvements in query performance."

**Comment by kylebarron** [comment]
FlatGeobuf's spatial index is always sorted on the geometries' hilbert value, so it doesn't matter what order the geometries are when passed into the FlatGeobuf writer.

**Reply by mrmichaelf** [reply]
The purpose of my comment is to shade a light about misleading documentation: One can understand "sorting input is important", while in reality "ordering your rows randomly will result... improvements in ... performance".

**Reply by kylebarron** [reply]
The implementations of the PostGIS R-Tree and FlatGeobuf's hilbert-packed R-Tree are different. 

Maybe we should take out the reference to that blog post.
